registrations/templates/base_email.html

```diff
@@ -0,0 +1,311 @@
+        <tr>
+          <td class="header-cell" style="padding:0;font-weight:normal;margin:0;text-align:left;padding:32px 16px;">
+            <a href="{{linked_events_ui_url}}/fi" target="_blank" class="brand-name" style="text-decoration:none;color:#1a1a1a;font-size:24px;font-weight:700;">
+              <img alt="" aria-hidden="true" src="{{linked_events_ui_url}}/images/email/logo.png" style="height:60px;width:130px;margin-right:16px;vertical-align:top;" />
+              <span style="line-height:52px;">Linked Events</span>
+            </a>
+          </td>
+        </tr>
+        {% block content %}
+        {% endblock content %}
+        <tr>
+          <td>
+            <footer class="footer" style="margin-top:32px;">
+              <section>
+                <div class="koro" style="height:30px;background-size:200px 30px;background-repeat:repeat-x;background-image:url({{linked_events_ui_url}}/images/email/koro-light.png);"></div>
+                <div class="footer-content" style="background-color:#f2f2f2;padding:32px 16px;">
+                  <div class="footer-title">
+                    <img alt="" aria-hidden="true" src="{{linked_events_ui_url}}/images/email/logo.png" style="height:40px;width:87px;margin-right:16px;display:inline-block;vertical-align:top;" />
+                    <h2 style="color:#1a1a1a;font-size:20px;font-weight:700;line-height:36px;margin:0px;display:inline-block;margin:0px;">Linked Events</h2>
+                  </div>
+                  <div class="footer-divider" style="background:#1a1a1a;border-radius:0px;height:1px;width:100%;margin:24px 0;"></div>
+                  <div class="footer-utilities" style="display:table;width:100%;direction:rtl;">
+                    <div class="footer-navigation" style="display:table-cell;vertical-align:top;direction:ltr;text-align:left;">
+                      <a href="mailto:linkedevents@hel.fi" style="color:#1a1a1a;font-size:16px;font-weight:normal;text-decoration:none;line-height:24px;margin-right:24px;">{% blocktranslate %}Ota yhteyttä{% endblocktranslate %}</a>
+                      <a target="_blank" href="{{linked_events_ui_url}}/fi/help/support/contact" style="color:#1a1a1a;font-size:16px;font-weight:normal;text-decoration:none;line-height:24px;">{% blocktranslate %}Anna palautetta{% endblocktranslate %}</a>
+                    </div>
+                  </div>
+                  <div class="footer-divider" style="background:#1a1a1a;border-radius:0px;height:1px;width:100%;margin:24px 0;"></div>
+                  <div class="copyright" style="margin-bottom:12px;">
+                    <span style="font-size:14px;margin-right:4px;">© {% blocktranslate %}City of Helsinki 2023{% endblocktranslate %}</span>
+                    <span class="copyright-dot" style="font-size:14px;margin-right:4px;">•</span>
+                    <span style="font-size:14px;">{% blocktranslate %}All rights reserved{% endblocktranslate %}</span>
+                  </div>
```

Comment:
More of a question than a request for change, but should we the string in Finnish on this copyright div? I assume the translations will be done later? :thinking:

Reply:
Good catch! I agree that all strings should be in Finnish. Fixed.
The translations will be done later. Registration part of LE API is missing quite many translations and it would be nice to add them all at once